core/arch/x86/decode_table.c

```diff
@@ -2936,9 +2936,9 @@ const instr_info_t prefix_extensions[][12] = {
     {OP_cvtpi2pd, 0x660f2a10, "cvtpi2pd", Vpd, xx, Qq, xx, xx, mrm, x, END_LIST},
     {OP_cvtsi2sd, 0xf20f2a10, "cvtsi2sd", Vsd, xx, Ed_q, xx, xx, mrm, x, END_LIST},
     {INVALID,  0x0f2a10, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {OP_vcvtsi2ss, 0xf30f2a10, "vcvtsi2ss", Vss, xx, H12_dq, Ed_q, xx, mrm|vex, x, END_LIST},
+    {OP_vcvtsi2ss, 0xf30f2a10, "vcvtsi2ss", Vdq, xx, H12_dq, Ed_q, xx, mrm|vex, x, END_LIST},
```

Comment:
I checked the manual and confirmed the extra-source version does write the full dest.

Did you check other similar-pattern opcodes (e.g., movss vs vmovss) to look for any other similar issues?

Reply:
I hadn't. Indeed, found 2 more bugs (or 4 incl evex). No guarantee for completeness. Primarily popping up when adding evex entries.